src/styles/blocks/main.scss

```diff
@@ -0,0 +1,40 @@
+.main {
+  padding: 50px 40px;
```

Comment:
Consider using relative units (like percentages or ems) for padding to improve responsiveness.